internal/exec/vendor_component_utils.go

```diff
@@ -50,12 +63,9 @@ func ReadAndProcessComponentVendorConfigFile(
 		return componentConfig, "", fmt.Errorf("folder '%s' does not exist", componentPath)
 	}
 
-	componentConfigFile := path.Join(componentPath, cfg.ComponentVendorConfigFileName)
-	if !u.FileExists(componentConfigFile) {
-		return componentConfig, "", fmt.Errorf("component vendoring config file '%s' does not exist in the '%s' folder",
-			cfg.ComponentVendorConfigFileName,
-			componentPath,
-		)
+	componentConfigFile, err := findComponentConfigFile(componentPath, strings.TrimSuffix(cfg.ComponentVendorConfigFileName, ".yaml"))
```

Comment:
Not the cleanest way to accomplish this but did not feel the liberty to edit the constants